docs/source/kedro_project_setup/dependencies.md

```diff
@@ -5,21 +5,29 @@ Both `pip install kedro` and `conda install -c conda-forge kedro` install the co
 When you create a project, you then introduce additional dependencies for the tasks it performs.
 
 ## Project-specific dependencies
-You can use Kedro to specify a project's exact dependencies to make it easier for you and others to run your project in the future, and to avoid version conflicts downstream.
+You can specify a project's exact dependencies in the `src/requirements.txt` file to make it easier for you and others to run your project in the future, 
+and to avoid version conflicts downstream. This can be achieved with the help of [`pip-tools`](https://pypi.org/project/pip-tools/). 
+To install `pip-tools` in your virtual environment, run the following command:
+```bash
+pip install pip-tools
+```
 
 To add or remove dependencies to a project, edit the `src/requirements.txt` file, then run the following:
 
 ```bash
-kedro build-reqs
+pip-compile --output-file=<project_root>/src/requirements.txt --input-file=<project_root>/src/requirements.txt
 ```
 
-The `build-reqs` command will [pip compile](https://github.com/jazzband/pip-tools#example-usage-for-pip-compile) the requirements listed in the `src/requirements.txt` file into a `src/requirements.lock` that specifies a list of pinned project dependencies (those with a strict version).
+This will [pip compile](https://github.com/jazzband/pip-tools#example-usage-for-pip-compile) the requirements listed in 
+the `src/requirements.txt` file into a `src/requirements.lock` that specifies a list of pinned project dependencies 
+(those with a strict version). You can also use this command with additional CLI arguments such as `--generate-hashes`. 
```

Comment:
What's the purpose of `--generate-hashes`? I think it's worth to mention what that's for, or just point people to the docs for all the other CLI argument options.

Reply:
Thanks @merelcht! I've updated `deployment/distributed.md` page and added more information about the CLI arguments.